pages/education-and-advocacy.md

```diff
@@ -89,28 +89,28 @@ Chasm” model and Albert Wong’s framework image._
 
 Fulfilling each step is necessary to producing lasting, positive change. By
 moving from one step to the next (conceptually; in reality many efforts must
-happen in parallel), the Early Majority is transformed from “Dependent” upon
-the Instigators to “Independent”:
+happen in parallel), the Early Majority is transformed from dependent upon
```

Comment:
"Dependent" and "Independent" here were referring to the labels on the Rainbow of Death graphic. Not sure if they should stay as they were, or as you've updated them here.